### PR TITLE
install kata custom version

### PR DESCRIPTION
**Pull request by ebattat**
Fix:
Add ability to install custom kata version instead of latest version. [Required when there is an issue in latest version]

### CODE REVIEW

File: .github/workflows/Weekly_Func_Env_Operator_CI.yml [hunk 1]
```diff
@@ -80,6 +80,8 @@ jobs:
           CNV_VERSION: "4.13"
           ODF_VERSION: "4.12"
           NUM_ODF_DISK : "4"
+          KATA_CHANNEL: "stable-1.4"
```
Comment (RobertKrawitz) [inline .github/workflows/Weekly_Func_Env_Operator_CI.yml:83]: Need a comment that this is a temporary workaround here and elsewhere.

**Review by RobertKrawitz — CHANGES_REQUESTED** [review]
The PR text needs to note that this is a short-term workaround, and this needs to be reverted (at least the places that set it) as soon as the underlying problem is fixed.

File: .github/workflows/Weekly_Func_Env_Operator_CI.yml (hunk 1 above)
Comment (ebattat) [reply .github/workflows/Weekly_Func_Env_Operator_CI.yml:83]: Its not temporary, once 1.4.1 issue fix I will remove just the value and I will leave all the rest as is

Comment (RobertKrawitz) [reply .github/workflows/Weekly_Func_Env_Operator_CI.yml:83]: Discussed offline and agree with the current wording.

**Review by RobertKrawitz — APPROVED** [review]

**Pull request merged**